docs/developer/index.rst

```diff
@@ -15,3 +14,17 @@ Version number
 The version number is currently stored in 3 places::
 
     PIL/__init__.py _imaging.c setup.py
+
+
+Readthedocs
+~~~~~~~~~~~
+
+Make sure the default version for Readthedocs is the latest release version e.g. 3.0.0 not latest.
```

Comment:
K, moved to RELEASING & added link to https://readthedocs.org/dashboard/pillow/versions/